operations/kube-manifests/ConfigMap-tempo-querier.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
```

Comment:
Same here, this looks more than querier?

Reply:
Oh right, at the moment we load almost the entire config into all components. This is actually the correct generated configmap.

Reply:
We should clean it up eventually. But notice that this doesn't have the compactor config block (its marked null)

Reply:
@bwplotka - are we good to proceed?

Reply:
Sorry for week lag, those are essentially different configs people can use? 🤔 How one can tell what we should use? (: